server/index.js

```diff
@@ -17,8 +17,31 @@ app.listen(port, () => {
 app.get('/', (req, res) => {
   res.sendFile(path.join(__dirname, '../client/dist/index.html'));
 });
-
-// app.get('/cart', (req, res) => {
-//   res.sendStatus(200);
-//   db.fetch(res.send.bind(res));
-// });
+// gets cart contents
+app.get('/carts', (req, res) => {
+  db.mySqlConnection.query('SELECT * FROM cart', (error, results) => {
+    if (error) {
+      throw error;
+    }
+    res.send(results);
+  });
+});
+// testing cart post/get
+app.post('/carts', (req, res) => {
+  db.mySqlConnection.query("INSERT INTO cart(user_id, item_id) VALUES ('1','5' )", (err) => {
+    if (err) {
+      throw err;
+    }
+  });
+  console.log('we did it');
```

Comment:
I'm not sure if this was intentional but don't **lines 36-37** (`console.log('we did it')` and `res.send(200)`) belong inside the callback of the database query if it's successful? You might want to consider changing this later on in case there is something wrong with the query.